FeedAPIChallenge/Feed API/Helper/FeedImageMapper.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Carlos Marcelo Tonisso Junior on 02/04/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImageMapper {
+	
+	private struct Root: Codable {
+		internal var items: [Item]
+		internal var images: [FeedImage] {
+			return items.compactMap({ $0.item })
```

Comment:
No need to `compactMap` since there are no optional values to discard. Use `map` instead.

```suggestion
			return items.map(\.item)
```